.github/workflows/CI.yml

```
@@ -151,9 +151,7 @@ jobs:
         run: |
           PYTEST_ARGS+=" --ignore=openff/toolkit/tests/test_examples.py"
           PYTEST_ARGS+=" --ignore=openff/toolkit/tests/test_links.py"
-          if [[ "$GITHUB_EVENT_NAME" == "schedule" ]]; then
-            PYTEST_ARGS+=" --runslow"
-          fi
+          PYTEST_ARGS+=" --runslow"
```

Let's just be sure to revert this before merging!
```suggestion
          if [[ "$GITHUB_EVENT_NAME" == "schedule" ]]; then
            PYTEST_ARGS+=" --runslow"
          fi
```